lib/dotcom_web/components/components.ex

```diff
@@ -229,4 +229,38 @@ defmodule DotcomWeb.Components do
     </details>
     """
   end
+
+  slot(:inner_block, required: true)
+  attr(:class, :string, default: "")
+
+  attr(:placement, :atom,
+    default: :left,
+    values: [:left, :right, :top, :bottom],
+    doc: "Where the tooltip appears relative to the hovered content."
+  )
+
+  attr(:title, :string,
+    doc:
+      "A preferably short bit of content which will appear on hover. Supports rendering of HTML."
+  )
+
+  @doc """
+  Basic tooltip to display content on hover.
+
+  This uses Bootstrap's JS plugin under the hood.
```

Comment:
Is there an alternative to Bootstrap that we can easily plugin? Just thinking that we want to remove Bootstrap so this is adding another place we would eventually have to remove it.

Reply:
> easily

No. Tooltips are one of the trickier things to get right, accessibility-wise. 

Part of my thinking is that, at the very least, the more places that use this reusable component, the fewer spots across our templates we'll have to touch to migrate from Bootstrap later. Keep in mind this isn't adding new Bootstrap tooltips where there isn't, just swapping out the template.